src/ocrd_utils/os.py

```diff
@@ -151,7 +151,7 @@ def list_all_resources(executable, moduled=None, xdg_data_home=None):
         if datadir.is_dir():
             candidates += datadir.iterdir()
     if 'system' in resource_locations:
-        systemdir = Path('/usr/local/share/ocrd-resources', executable)
+        systemdir = Path(RESOURCES_DIR_SYSTEM, executable)
         if systemdir.is_dir():
             candidates += systemdir.iterdir()
     if 'module' in resource_locations and moduled:
```

Comment:
btw, we should include `'*.egg-info/*', 'copyright.txt', 'LICENSE*', 'README.md',` in the list of anti-patterns below